tensorboard/util.py

```diff
@@ -419,7 +419,8 @@ def _lazily_initialize(self):
       graph = tf.Graph()
       with graph.as_default():
         self.initialize_graph()
-      self._session = tf.Session(graph=graph)
+      config = tf.ConfigProto(device_count={'GPU': 0})
```

Comment:
Maybe add a comment here noting that we're deliberately keeping this off the GPU?

Reply:
Done